src/fluree/db/storage/s3.clj

```diff
@@ -325,13 +323,13 @@
     ch))
 
 (defn s3-address
-  [identifier s3-bucket s3-prefix path]
-  (storage/build-fluree-address identifier method-name path [s3-bucket s3-prefix]))
+  [path]
```

Comment:
The identifier allows users to use two different s3 storage instances on the same connection. Without the identifier, there would be no way to differentiate them.